code/modules/shuttle/shuttle.dm

```diff
@@ -606,11 +610,13 @@
 				mode = SHUTTLE_CALL
 				setTimer(callTime * engine_coeff)
 				enterTransit()
+				SEND_SIGNAL(src, COMSIG_SHUTTLE_CALL, destination)
 				return
 
 	mode = SHUTTLE_IDLE
 	timer = 0
 	destination = null
+	SEND_SIGNAL(src, COMSIG_SHUTTLE_IDLE)
```

Comment:
Having this on a `set_shuttle_idle()` or `set_shuttle_status(SHUTTLE_IDLE)` proc might be better.

Reply:
No that would cause infinite recursion if anything registered to this signal called those procs